rwandaemr-installer/src/main/resources/scripts/download-distribution.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/bash -eu
 
-ARTIFACT_ID="$1"
-VERSION="$2"
-TARGET_DIR="$3"
+## load variables from the vars.txt file
+source vars.txt
+
+ARTIFACT_ID="${distribution}"
+VERSION="${version}"
+
+if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
+  TARGET_DIR="/opt/${parent_dir}/distribution"
+else
+  TARGET_DIR="${parent_dir}/distribution"
+fi
```

Comment:
Set this target directory at a top level somewhere.  Put it in vars.txt or whatever, based on the input ${parent_dir}, then you can reliably refer to it in any downstream scripts.